DetectionFactory/DetectionFactory.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
+
+        <IsPackable>false</IsPackable>
+
+        <RootNamespace>FactoryDetectionTests</RootNamespace>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="FluentAssertions" Version="6.0.0-beta0001" />
+        <PackageReference Include="NUnit" Version="3.12.0" />
+        <PackageReference Include="NUnit3TestAdapter" Version="3.16.1" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\src\Business\Business.csproj" />
+      <ProjectReference Include="..\src\Data\Data.csproj" />
+    </ItemGroup>
+
+</Project>
```

Comment:
Should we rename this project into: `DataTests` one?

Reply:
Should we move this project under the `test` folder? Should we make sure we use solution folders also?

Reply:
Done